packages/boot/test/bootstrapTests/benchmarkerator.md

```diff
@@ -0,0 +1,171 @@
+# The Benchmarkerator
```

Comment:
This readme made understanding and reviewing the tool a breeze. Appreciate the effort you put into it!